src/index.js

```diff
@@ -7,7 +7,7 @@ const defaultPosition = 'bottom'
 const prepareBinding = ({ arg = '', modifiers = {}, value = {} }) => {
   let mods = Object.keys(modifiers)
   let name = typeof value === 'object' && value.name ? value.name : arg
-  let position = mods[0] || defaultPosition
+  let position = mods[0] || value.position || defaultPosition
```

Comment:
Good call. but i think you will have to check if `value` is object (look line 9). If you will fix this i am happy to merge your changes.

THanks a lot for PR btw :)

Reply:
@euvl Are you supporting passing any other type? Since the default value for the variable "value" (:sweat_smile: ) is `{}` I thought it would always be like that.
I guess it could throw error if `value === null`, which is only possible if the user writes `v-popover="null"`. Otherwise, I think `value.position` will be `undefined`.

If you still want it I can check `typeof value === 'object'`, of course :+1:

Reply:
Seems that i were using String value for creating tooltip:

https://github.com/euvl/vue-js-popover/blob/d00a45069a95f21ddfd26502f46adc5b2c0ee2a9/demo/src/App.vue#L17

Reply:
Oh, the tooltip! `('string').position` is still undefined like `({}).position`, but I understand you prefer it explicit. I'll change it tomorrow, time to sleep now :)

Reply:
Cool, ty